detox/test/android/app/src/main/java/com/example/NativeModule.java

```diff
@@ -111,42 +110,40 @@ public void getLaunchArguments(Promise promise) {
 
     @ReactMethod
     public void spyLongTaps(final String testID) {
-        new LongTapsSpy(testID).spyOnce();
+        new LongTapCrasher(testID).attach();
     }
 
     /**
      * Implementation note: For this purpose, a simpler RN API exists in the same class -
      * {@link ReactFindViewUtil#addViewListener(ReactFindViewUtil.OnViewFoundListener)}.
-     * However, it is found to be a but buggy since it removes all listeners immediately
+     * However, it is found to be a bit buggy since it removes all listeners immediately
      * after being called (i.e. while iterating) with no thread-sync mechanisms to protect it.
-     * If real life (CI), we've genuinely seen on occasions that it throws ConcurrentModificationException
-     * exceptions (and why wouldn't it? - we have a screen with multiple views subscribing and
-     * called concurrently in the ActionsScreen of this demo app; could it be that it is not always called
-     * from the main thread?).
+     * If real life (CI), we've genuinely seen that it throws ConcurrentModificationException exceptions,
+     * on occasions (and why wouldn't it? - our demo app's ActionsScreen has multiple views subscribing and
+     * called concurrently; could it be that not always everything is run in the main thread?).
      * Therefore, we use here {@link ReactFindViewUtil#addViewsListener(OnMultipleViewsFoundListener, Set)},
      * which is too generic but nevertheless allows us to better control when we are to be removed.
      */
-    private static class LongTapsSpy implements OnMultipleViewsFoundListener {
+    private static class LongTapCrasher implements OnMultipleViewsFoundListener {
```

Comment:
It does not. It's here because the branch was merged-to from master instead of rebasing it @valentynberehovyi ❤️

Reply:
@valentynberehovyi you can still solve this by rebasing the branch. In the future, let's keep this in mind as the reason behind why we always rebase rather than merge 😄 
@rotemmiz #not_nitpickying_anymore